SchedulerBot/SchedulerBot.Database.Core/Migrations/20180430150138_ManageConversationLink.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using System;
+
+namespace SchedulerBot.Database.Core.Migrations
+{
+	public partial class ManageConversationLink : Migration
+	{
+		protected override void Up(MigrationBuilder migrationBuilder)
+		{
+			migrationBuilder.CreateTable(
+				name: "ManageConversationLinks",
+				columns: table => new
+				{
+					Id = table.Column<Guid>(nullable: false),
```

Comment:
Should be there some autogeneration parameter? like `auto: true` or something like that

Reply:
It's in `ManageConversationLink.Designer.cs` file, see line #&#xfeff;23:

```cs
b.Property<Guid>("Id")
	.ValueGeneratedOnAdd();
```